### ISSUE TITLE
Field ip_filters in VMauth is missing

### ISSUE THREAD

**Issue by tropnikovvl** [VictoriaMetrics/operator#1168]
Hi.
According to [this](https://docs.victoriametrics.com/operator/resources/vmauth/#ip-filters) documentation, **unauthorizedAccessConfig** should contain **ip_filters**

```yaml
  unauthorizedAccessConfig:
    - src_paths: ["/metrics"]
      url_prefix: ["http://vmsingle-example.default.svc:8428"]
      ip_filters:
        allow_list:
          - 192.168.0.0/16
          - 10.0.0.0/8
```

but this field is not [here](https://docs.victoriametrics.com/operator/api/index.html#unauthorizedaccessconfigurlmap) and it is also not in the CRD.

**Comment by f41gh7** [comment]
Hello, 
 Thanks for reporting! Given configuration example is incorrect, we'll fixed that.
 
It seems, that for now it's not possible to define `ip_filters` for  `unauthorizedAccessConfig`. It will require to add new setting `unauthorizedAccessSettings` to the `VMAuth.spec` with configuration settings for `unauthorizedAccessConfig`, to reflect vmauth config params:
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/app/vmauth/auth_config.go#L68-L82

**Comment by f41gh7** [comment]
The issue was fixed at [v0.51.1](https://github.com/VictoriaMetrics/operator/releases/tag/v0.51.1) release
